.github/workflows/bump-ui-etl-version.yml

```
@@ -0,0 +1,447 @@
+        default: false
+
+# Serialize: release.py can post several statuses in one run, and a re-dispatch
+# can overlap a real release. Two runs must not race on the bump branch, the PR,
+# or the failure issue. Never cancel - a dropped run means a missed bump/report.
+concurrency:
+  group: bump-ui-etl-version
+  cancel-in-progress: false
+  # Keep up to 100 runs pending and process them in order. The default (single)
+  # cancels the pending run whenever a new one arrives - which is exactly the
+  # burst release.py produces when it posts several statuses for one release.
+  queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)